source/common/http/conn_manager_impl.cc

```diff
@@ -335,6 +340,19 @@ Network::FilterStatus ConnectionManagerImpl::onData(Buffer::Instance& data, bool
   return Network::FilterStatus::StopIteration;
 }
 
+Network::FilterStatus ConnectionManagerImpl::onNewConnection() {
+  if (!read_callbacks_->connection().streamInfo().protocol()) {
+    return Network::FilterStatus::Continue;
```

Comment:
Presumably we expect this to be a terminal filter so this should stop iteration as you have done below? (Edit I see you copied this from the header, but let's make it stop to be consistent).

Reply:
done

Reply:
OK that's fine. Can you add a comment so the reader understands why we use continue in one place and stop iteration in another?